spec/integration/dry/rails/container/auto_register/with_block_spec.rb

```diff
@@ -5,8 +5,12 @@
 
   before(:all) do
     Dry::Rails.container do
-      auto_register!("app/workers") do |config|
-        config.memoize = true
+      begin
+        config.component_dirs.add("app/workers") do |dir|
```

Comment:
it's indeed redundant, but dunno, if we have multiple `component_dirs.add` I feel like it's better to be explicit and wrap them separately - in that case we need `begin` to be explicit, so I wanted to use it consistently everywhere

is it okay to disable this cop?